### PR TITLE
Remove submodules and update development guide

### PR DESCRIPTION
**Pull request by Walingar**
Core repository can be used separately from main compose-multiplatform repository and vice versa. 
So, now we don't need submodules, but since our main development is going in core repository I highlighted it in README.md in compose folder.

### CODE REVIEW

**Comment by Walingar** [comment]
Regarding publishing, since it seems that compose-multiplatform should be linked with core repository, but it is not necessary. When we need to publish a new release version, we should create `release/x` branch and Teamcity will handle synchronization. Same for dev builds, Teamcity will handle synchronization and take the most recent changes from core and compose-multiplatform repositories

**Comment by dima-avdeev-jb** [comment]
I think we should wait @igordmn, because we have publishing scripts in this repository, like **publishExtendedIcons**.
We can accidentally break something on CI.

**Review by dima-avdeev-jb — APPROVED** [review]

File: compose/README.md [hunk 1]
```diff
@@ -1,100 +1,49 @@
...
-## Scripts
-Publish artifacts to the local directory `out/androidx/build/support_repo/org/jetbrains/compose`:
-```
-export COMPOSE_CUSTOM_VERSION=0.0.0-custom
-./scripts/publish
-```
-(on Windows it doesn't build at the moment, the issue is in the line `packageInspector(project, project(":compose:ui:ui-inspection")` in `compose/frameworks/support/compose/ui/ui/build.gradle`)
+Compose Multiplatform uses [Skiko](https://github.com/JetBrains/skiko). A low-level library that hides platform
+complexity and provides a simple interface for rendering, event handling, window management, and other features. Skiko
+uses Skia as graphical API.
```
Comment (MatkovIvan) [inline compose/README.md:15]: Should we point to our fork of skia?

File: compose/README.md [hunk 2]
```diff
@@ -1,100 +1,49 @@
...
-## Publish additional libraries to mavenLocal
+4. Publish additional components
 ```bash
-export COMPOSE_CUSTOM_VERSION=0.0.0-custom-version &&\
-./scripts/publishWebComponentsToMavenLocal &&\
-./scripts/publishExtendedIconsToMavenLocal -Pcompose.platforms=all &&\
 ./scripts/publishComponentsToMavenLocal
 ```
+
+5. Publish additional Web components
```
Comment (eymar) [inline compose/README.md:46]: Let's call it Compose HTML library here.

Also the script can be renamed.
Plus, the script itself needs to be updated:
https://github.com/JetBrains/compose-multiplatform/blob/master/compose/scripts/publishWebComponentsToMavenLocal

`pushd ../../web` -> `pushd ../../html`

If it's okay to do this change in this PR

Comment (Walingar) [reply compose/README.md:46]: I think it is ok, will do!

File: compose/README.md (hunk 1 above)
Comment (Walingar) [reply compose/README.md:15]: Not sure, from my point it should be handled in skiko

**Comment by Walingar** [comment]
> I think we should wait @igordmn, because we have publishing scripts in this repository, like publishExtendedIcons.
We can accidentally break something on CI.

Discussed with Dima that it is ok to merge it now, since CI will be changed and I'll ask Igor to review these CI changes

File: compose/README.md (hunk 2 above)
Comment (Walingar) [reply compose/README.md]: Done

**Review by eymar — APPROVED** [review]

**Pull request merged**